src/applications/my-education-benefits/utils/form-submit-transform.js

```diff
@@ -289,7 +289,9 @@ export function createContactInfo(submissionForm) {
     addressLine1: submissionForm['view:mailingAddress'].address.street,
     addressLine2: submissionForm['view:mailingAddress'].address.street2,
     city: submissionForm['view:mailingAddress'].address.city,
-    zipcode: submissionForm['view:mailingAddress'].address.postalCode,
+    zipcode:
+      submissionForm['view:mailingAddress'].address.postalCode ||
+      submissionForm['view:mailingAddress'].address.internationalPostalCode,
```

Comment:
Be careful with this logic. If you enter a postal code, change the country to be non-US (or whatever the logic is for showing the international postal code field) and fill out the international postal code, the original postal code will still be set. So, this will submit the regular postal code and not the international postal code.

Reply:
Got it, reverted back to original postal code

Reply:
Okay, changes look good.